frontend/src/container/GuestUpload.jsx

```diff
@@ -36,6 +38,27 @@ class GuestUpload extends React.Component {
     }
   }
 
+  async componentDidMount () {
+    const { props } = this
+    const request = await getGuestUploadInfos(props.match.params.token)
+    const response = await handleFetchResult(request)
```

Comment:
if getGuestUploadInfos() properly uses the fetchWrapper (as asked above), there is no need to call handleFetchResult()